### PR TITLE
Add startrek Trivia list

### PR DESCRIPTION
**Pull request by scarecr0w12**
Star trek trivia list

### Type

- [ ] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes
Star trek trivia list

### CODE REVIEW

**Comment by Vexed01** [comment]
Please note that the bot is case sensitive with answers. As an example in [line 5](https://github.com/Cog-Creators/Red-DiscordBot/blob/47dc1caf61a164fcbe82cd291813bceeb884ee6a/redbot/cogs/trivia/data/lists/startrek.yaml#L5) you put the single answer as `pennsylvania`. This means that the bot will mark `Pennsylvania` incorrect. That question would ideally look like this:
```
In What State Is Carbon Creek Located?:
- pennsylvania
- Pennsylvania
```

**Comment by scarecr0w12** [comment]
Yeah i realized that after i got it set, im gonna go through the list again and add in alternative forms of the answers and capitalization. Wanted to get a first set up though.

File: redbot/cogs/trivia/data/lists/startrek.yaml [hunk 1]
```diff
@@ -0,0 +1,3159 @@
...
+What Type Of Game Did Maggie Take Mestral To See?:
+- baseball
+Where Did The Vulcans Make Official First Contact With Zefram Cochrane?:
+- bozeman, Montana
+What Was Mestral Watching On The Tv In The Bar?:
+- atomic Bomb Test
+What Species Forwarded The Distress Signal To The Vulcan High Command?:
+- tellarite
+What Is A Vulcan Second Foremother?:
+- a Mother?s Mother?s Mother
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/startrek.yaml]: Looks like a `'` got turned into a `?` here.
```suggestion
- a Mother's Mother's Mother
```

**Review by Flame442 — CHANGES_REQUESTED** [review]
If you're still interested in working on this (sorry for the 3 year wait), the questions should have normal English capitalization rules for a sentence (only capitalize the first letter and proper nouns) and a lot of these questions could use alternative answers, especially the dates and names, to accept a bit more variety in how a user could input the answer. Note that trivia **is** case insensitive for answers, so you **do not** need to add different *casings*, only different *phrasings* (`august 19, 1921` could also be `8/19/1921` or `19/8/1921`).

File: redbot/cogs/trivia/data/lists/startrek.yaml (hunk 1 above)
Comment (scarecr0w12) [reply redbot/cogs/trivia/data/lists/startrek.yaml]: Totally can, Kinda forgot about this over time, but i can finish this out, no problem.

Comment (aikaterna) [reply redbot/cogs/trivia/data/lists/startrek.yaml]: > Totally can, Kinda forgot about this over time, but i can finish this out, no problem.

Hi there, I don't want this to sound like I'm pushing you but if you don't have the time for this I can get it wrapped up and approved, just let me know and I'll be happy to reformat and help. Please let us know soonish if you can, this item is slated to hopefully be included with the next update if everything goes well. Again sorry for being a bit pushy about time when this has been sitting for so long.

**Comment by aikaterna** [comment]
This is ready for review, if there are any Star Trek nerds out there that would like to go over the changes to this file, besides my review request for Flame.

As a comment on this file for the future, the majority of the trivia questions are from Star Trek: The Original Series with a handful from The Next Generation, Voyager, Deep Space 9, and some of the older movies. If someone wanted to expand this list or add to it, I would suggest more entries from other parts of the Star Trek universe or spinoffs instead of the Original Series.

File: redbot/cogs/trivia/data/lists/startrek.yaml [hunk 2]
```diff
@@ -0,0 +1,1975 @@
...
+- Bozeman, Montana
+What was Mestral watching on the TV in the bar?:
+- Atomic bomb test
+- bomb test
+- atomic test
+What species forwarded the distress signal to the Vulcan High Command?:
+- Tellarite
+What is a Vulcan second foremother?:
+- A mother's mother's mother
+What does Mestral use to free the trapped minors?:
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/startrek.yaml]:
```suggestion
What does Mestral use to free the trapped miners?:
```

File: redbot/cogs/trivia/data/lists/startrek.yaml [hunk 3]
```diff
@@ -0,0 +1,1975 @@
...
+- Qpid
+Who testified that the prohibited maneuver, the Kovoord Starburst, was the cause of cadet Joshua Albert's death?:
+- Wesley Crusher
+"What was the green liquid that Data gave to Scotty in the Star Trek: The Next Generation episode, Relics?":
+- Aldebaran whiskey
+"What is the name of the shape-shifting species from the Star Trek: The Next Generation episode The Dauphin?":
+- Allasomorph
+In what episode was Captain Picard kidnapped by aliens and put into a room with other humans to study their behavior?:
+- Allegiance
+"In the Star Trek: The Original Series episode The Ultimate Computer, what planet did the M-5 computer scan to test it's performance with planetary contact?":
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/startrek.yaml]:
```suggestion
"In the Star Trek: The Original Series episode The Ultimate Computer, what planet did the M-5 computer scan to test its performance with planetary contact?":
```

File: redbot/cogs/trivia/data/lists/startrek.yaml [hunk 4]
```diff
@@ -0,0 +1,1975 @@
...
+- Best of Both Worlds
+On what stardate did the Borg capture Captain Picard and then attempt to assimilate Earth?:
+- 43989.1
+"What is the name of Deanna Troi's home planet?":
+- Betazed
+"What kind of metal is used in Data's neural pathways?":
+- Bitanium
+What is the most sacred planet in the Klingon empire, the supposed returning point of Kahless?:
+- Boreth
+What race uses a huge cube-shaped craft with a higly decentralized design?:
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/startrek.yaml]:
```suggestion
What race uses a huge cube-shaped craft with a highly decentralized design?:
```

File: redbot/cogs/trivia/data/lists/startrek.yaml [hunk 5]
```diff
@@ -0,0 +1,1975 @@
...
+- Elizabeth Shelby
+- Lieutenant Commander Shelby
+- Lt. Commander Shelby
+What name did the Borg give Captain Picard after assimilating him into their collective?:
+- Locutus
+How many ships of the armada at Wolf 359 were destroyed by the Borg?:
+- 39
+How many Federation lives were lost at the Battle of Wolf 359?:
+- 11,000
+What was the name of the Borg that survivied the crash of his scout ship, who was captured by Enterprise-D?:
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/startrek.yaml]:
```suggestion
What was the name of the Borg that survived the crash of his scout ship, who was captured by Enterprise-D?:
```

File: redbot/cogs/trivia/data/lists/startrek.yaml [hunk 6]
```diff
@@ -0,0 +1,1975 @@
...
+- All Good Things...
+- All Good Things
+"Who lost his life at the Starfleet Academy flight range near Saturn in 2368 in the Star Trek: The Next Generation episode, The First Duty?":
+- Joshua Albert
+- Josh Albert
+"In the original proposal for The Next Generation, what was Captain Picard's first name going to be?":
+- Julien
+The Next Generation proposal originally called for a female character in place of Wesley Crusher. What was her name?:
+- Leslie
+Which Next Generation episode was originally banned in Britain due to it's reference to terrorism and the IRA?:
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/startrek.yaml]:
```suggestion
Which Next Generation episode was originally banned in Britain due to its reference to terrorism and the IRA?:
```

**Review by Flame442 — CHANGES_REQUESTED** [review]

**Review by Flame442 — APPROVED** [review]

**Pull request merged**